From 778fc8e69bd57f557fe2ba4aec854d2bbdc8456f Mon Sep 17 00:00:00 2001
From: Niels Peter Roest <nero@jobindex.dk>
Date: Wed, 11 Mar 2026 16:05:49 +0100
Subject: [PATCH 3/3] Dont score lists negatively

Jobads often have lists, so these aren't an indicator of bad content.
---
 parser.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/parser.go b/parser.go
index d8fb249..18392a2 100644
--- a/parser.go
+++ b/parser.go
@@ -656,7 +656,7 @@ func (ps *Parser) initializeNode(node *html.Node) {
 		contentScore += 5
 	case "pre", "td", "blockquote":
 		contentScore += 3
-	case "address", "ol", "ul", "dl", "dd", "dt", "li", "form":
+	case "address", "form":
 		contentScore -= 3
 	case "h1", "h2", "h3", "h4", "h5", "h6", "th":
 		contentScore -= 5
-- 
2.53.0

